content/news/033/index.md

```diff
@@ -530,38 +532,35 @@ your project in an ergonomic manner without enforcing any structure or pattern,
 always trying to stay out of your way. The idea is that you can use it as a
 foundation layer or backend for your next app, game engine, or game.
 
-The latest version [v0.3.0] comes with audio support for all platforms
-using as default backend [oddio] and [symphonia].
+The latest version [v0.3.0] comes with audio support for all platforms using as
+default backend [oddio] and [symphonia].
 
 [Notan]: https://github.com/Nazariglez/notan
 [v0.3.0]: https://github.com/Nazariglez/notan/releases/tag/v0.3.0
 [oddio]: https://github.com/Ralith/oddio
 [symphonia]: https://github.com/pdeljanov/Symphonia
 
-## Popular Workgroup Issues in Github
-
-<!-- Up to 10 links to interesting issues -->
-
 ## Other News
 
 <!-- One-liners for plan items that haven't got their own sections. -->
 
 - Other game updates:
-  - [Last of the Sky Folk](https://ianjk.com/ld50)
-    is a LD50 a grapple-hook based platformer.
+  - [Last of the Sky Folk](https://ianjk.com/ld50) is a LD50 a grapple-hook
```

Comment:
Thanks for including this! 

This sentence seems to be missing a word. Here's a proposed fix:
 "[Last of the Sky Folk](https://ianjk.com/ld50) is a grapple-hook based platformer that's an entry for LD50."

Reply:
Fixed :) also, if you're interested in showing if off at tomorrow's gamedev meetup, feel free to fill out the form!

https://forms.gle/BS1zCyZaiUFSUHxe6